survey_formio/models/survey_question.py

```diff
@@ -0,0 +1,216 @@
+# Copyright 2023 Acsone SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+
+from odoo import models
+
+ODOO_TO_FORM_IO_TYPES_DICT = {
+    "free_text": "textarea",
+    "textbox": "textfield",
+    "numerical_box": "number",
+    "date": "day",
+    "datetime": "datetime",
+    "simple_choice": "radio",
+    "multiple_choice": "selectboxes",
+}
+
+
+class SurveyQuestion(models.Model):
+    _inherit = "survey.question"
+
+    def _get_common_parameters(self):
```

Comment:
When adding a lot of methods to an existing model, let's prefix them to avoid name conflicts and for clarify, so:

```suggestion
    def _get_formio_common_parameters(self):
```

Alternatively, put everything in a FormIOConverter abstract model with only model methods.

Reply:
I've added the prefix